net/squid4/Portfile

```diff
@@ -133,6 +133,22 @@ if {![variant_isset gnutls]} {
     default_variants    +openssl
 }
 
+variant sslbump conflicts gnutls description "Enable SSL Bump support" {
+    default_variants-append +openssl
+    configure.args-append   --enable-ssl-crtd
+    configure.args-append   --enable-http-violations
+    configure.args-append   --enable-security-cert-validators
+    configure.args-append   --enable-security-cert-generators
```

Comment:
These three are on by default. Also note you can add multiple things in one configure.args-append call.